<launch>

	<arg name="rviz" default="true" />
	<arg name="autorun"   default="false"/>
    <arg name="data_path" default="/media/gtx/D273-5B5F/Night_mapping/Collected_Dataset"/>
    <arg name="bag_file1"  default="$(arg data_path)/Localization_Sequences/Scene_04/Scene_04.bag"/>
	<arg name="bag_file2"  default="$(arg data_path)/Streetlight_Detections/Scene_04/Scene_04_Yolo.bag"/>

	<rosparam command="load" file="$(find night_voyager)/config/Config_scene_04.yaml" />

    <node pkg="night_voyager" type="night_voyager_node" name="night_voyager_node" output="screen" /> 

	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find night_voyager)/rviz/Night_Voyager.rviz"/>
    <!-- <param name="/use_sim_time" value="true" /> -->
    <node required="$(arg autorun)" pkg="rosbag" type="play" name="rosbag_player"
          args="--clock $(arg bag_file1) $(arg bag_file2) -r 1.0 -s 0"
          launch-prefix="bash -c 'sleep 1; $0 $@' "/>

</launch>
